### ISSUE TITLE
Multiple Sign-In Requests on Android when a configuration is provided?

### ISSUE THREAD

**Issue by DaveBigPixel** [playgameservices/play-games-plugin-for-unity#1122]
On Android, if I decline the first sign-in request popup (using the device 'back' button), I am prompted with another sign-in request a few seconds later - which is not desired.

This behaviour only occurs when PlayGamesPlatform.InitializeInstance(config) is called. commenting out the InitializeInstance line results in the correct behaviour but save games will not be enabled.

```
void Start()
    {
        // Config for Save Games
        PlayGamesClientConfiguration config = new PlayGamesClientConfiguration.Builder().EnableSavedGames().Build();

        // Init Play Games
        PlayGamesPlatform.InitializeInstance(config); // This causes an extra sign-in request if the first request is declined

        // recommended for debugging:
        PlayGamesPlatform.DebugLogEnabled = true;

        // Activate the Google Play Games platform
        PlayGamesPlatform.Activate();

        // Authenticate user
        Social.localUser.Authenticate(AuthHandler);
    }

    private void AuthHandler(bool success)
    {
        Debug.Log("AuthHandler(success:"+success+")");
    }
```

Any ideas?

**Comment by claywilkinson** [comment]
You can keep track of the "should authenticate" state in your app - so if the AuthHandler is called with success == false, keep that in a member variable, then don't call Authenticate again if the flag is set.

**Comment by DaveBigPixel** [comment]
The code I posted only calls Authenticate() once but it causes two sign in prompts if the first is dismissed.

I'm not calling Authenticate again.

**Comment by claywilkinson** [comment]
This is fixed in 0.9.39.
